status/controller.go

```diff
@@ -6,49 +6,159 @@ import (
 
 	"github.com/awslabs/operatorpkg/object"
 	"github.com/prometheus/client_golang/prometheus"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/client-go/tools/record"
 	controllerruntime "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+const (
+	MetricLabelGroup           = "group"
+	MetricLabelKind            = "kind"
+	MetricLabelNamespace       = "namespace"
```

Comment:
Drop label namespace and label name now that they are unused?

Reply:
restored these for object counts. Keeping it in.